File: introduction/ITP1_6_B-Finding Missing Cards.c
int main() {
  int n, i, j, d, cards[4][13] = {0};
  char s, suit[4] = "SHCD";
  scanf("%d\n", &n);
  for(i = 0; i < n; cards[s][d] = 1, i++)
    scanf("%c %d\n", &s, &d);
  for(i = 0; i < 4; i++)
    for(j = 1; j < 14; j++)
      if(!cards[suit[i]][j]) printf("%c %d\n", suit[i], j);
  return 0;
}

File: introduction/ITP1_6_B-Finding Missing Cards.cs
using System;
using System.Collections.Generic;

public class Program {
  public static void Main() {
    int n = int.Parse(Console.ReadLine());
    var cards = new List<string>();
    for(int i = 0; i < n; i++) cards.Add(Console.ReadLine());
    var suit = new List<string> {"S", "H", "C", "D"};
    for(int i = 0; i < 4; i++) {
      for(int j = 1; j < 14; j++) {
        string card = suit[i] + " " + j;
        if(!cards.Contains(card)) Console.WriteLine(card);
      }
    }
  }
}
